src/src_adjust_spatial_obs.py

```diff
@@ -74,7 +66,7 @@ def process_points(args):
     ## Define coords variable to be used in point raster value attribution.
     coords = [(x,y) for x, y in zip(water_edge_df.X, water_edge_df.Y)]
 
-    water_edge_df.to_crs(DEFAULT_FIM_PROJECTION_CRS)
+    water_edge_df = water_edge_df.set_crs(DEFAULT_FIM_PROJECTION_CRS, allow_override=True)
```

Comment:
`set_crs` assigns the crs without performing a transformation. I think we want to use `to_crs` to insure the input data gets converted to the `DEFAULT_FIM_PROJECTION` if needed. https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoDataFrame.set_crs.html

Reply:
@RyanSpies-NOAA - I must've mis-understood the docs when I made this change. Reviewing the docs again, you are correct. I reverted back to `.to_crs`, ran a `fim_pipeline` test with one HUC, and everything checks out. 

The projection of the `water_edge_df` is set correctly to the `DEFAULT_FIM_PROJECTION` as verified by: 
1.) Running this script with `-debug` option (which writes `water_edge_df` to a `.parquet` file)
2.) Loading the `.parquet` file as a GDF
3.) Checking that the projection of the GDF matches the `DEFAULT_FIM_PROJECTION`